NLog.Targets.OpenTelemetryProtocol/OtlpTarget.cs

```diff
@@ -34,6 +36,10 @@ public class OtlpTarget : TargetWithContext
 
         public bool IncludeFormattedMessage { get; set; }
 
+        public bool UseInjectedOptions { get; set; }
```

Comment:
Would remove this option, and only rely on whether the resolved option-name is not empty string.

Reply:
done